app/javascript/chat/ChatChannelSettings/ChatChannelSettings.jsx

```diff
@@ -305,6 +314,54 @@ export default class ChatChannelSettings extends Component {
     }
   };
 
+  toggleScreens = () => {
+    const { displaySettings, displayMembershipManager } = this.state;
+
+    this.setState({
+      displaySettings: !displaySettings,
+      displayMembershipManager: !displayMembershipManager,
+    });
+  };
+
+  handleUpdateMembershipRole = async (e) => {
+    const { membershipId, role } = e.target.dataset;
+    const { chatChannel } = this.state;
+    const response = await updateMembershipRole(
+      membershipId,
+      chatChannel.id,
+      role,
+    );
+    const { message } = response;
```

Comment:
Since `message` is being destructured, might as well destructure `success` as well.